templates/index.html

```diff
@@ -45,6 +45,39 @@
   </head>
 
   <body>
+    <!-- Release Note Popup -->
+    <div class="modal" tabindex="-1" role="dialog" id="release-note">
+      <div class="modal-dialog" role="document">
+        <div class="modal-content">
+          <div class="modal-header">
+              <h5 class="modal-title">April 12, 2021 Updates</h5>
+              <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                  <span aria-hidden="true">&times;</span>
+              </button>
+          </div>
+          <div class="modal-body">
+            <ul>
+                <li>Interactable object collision</li>
```

Comment:
Instead of the last update, let's instead preempt the next two changes:
- ramps
- preset fields (including autonomous portion of 2021 game field)